conduit/plugins/exporters/filewriter/README.md

```diff
@@ -55,12 +53,32 @@ You must use the proper expression type for the field your tag identifies based
 For example, do not use a numerical expression type on a string field such as address.
 
 
-#### `expression`
+### `expression`
 The expression is the data against which each field will be compared. This must be compatible with the data type of
 the expected field. For string fields you can also use the `regex` expression type to interpret the input of the
-expression as a regex. 
+expression as a regex.
+
+
+## Configuration
+```yml @sample.yaml
+  name: "file_writer"
+  config:
+    # BlocksDir is the path to a directory where block data should be stored.
+    # The directory is created if it doesn't exist. If no directory is provided
+    # blocks are written to the Conduit data directory.
+    #block-dir: "/path/to/block/files"
+
+    # FilenamePattern is the format used to write block files. It uses go
+    # string formatting and should accept one number for the round.
+    # If the file has a '.gz' extension, blocks will be gzipped.
+    # Default: "%[1]d_block.json"
+    filename-pattern: "%[1]d_block.json"
+
+    # DropCertificate is used to remove the vote certificate from the block data before writing files.
+    drop-certificate: true
+```
```

Comment:
this is a file_writer sample config but this readme is about transaction filtering?

Reply:
Thank you, fixed!